buildSrc/src/main/groovy/io.micronaut.build.internal.views-native-tests-library.gradle

```diff
@@ -1,6 +1,10 @@
 plugins {
     id "io.micronaut.minimal.library"
 }
+micronaut {
+    version.set(libs.versions.micronaut.platform.get())
```

Comment:
I am not sure that I understand why the micronaut version needs to be set to the `platform` version (4.0.3 instead of 4.1.0) when the platform bom is not used (disabled by `importMicronautPlatform = false`)?

Reply:
because there is no platform 4.1.0 yet

Reply:
The platform is not used at all since `importMicronautPlatform = false`. I am not against this change, just trying to understand how this is supposed to work since I will work on other tests too.

Reply:
the platform is not used but you have to define a valid version